Libplanet/Tx/RawTransaction.cs

```diff
@@ -180,14 +197,15 @@ private static T[][] To2dArray<T>(T[] array, int chunk)
             }
 
             int resultLength = array.Length / chunk;
-            T[][] result = new T[resultLength][];
+            ImmutableArray<T>[] result = new ImmutableArray<T>[resultLength];
             for (int i = 0; i < resultLength; i++)
             {
-                result[i] = new T[chunk];
-                Array.Copy(array, i * chunk, result[i], 0, chunk);
+                T[] partialResult = new T[chunk];
```

Comment:
We're using `var` when variable's type is obvious. (e.g. `var x = new T();`)

```suggestion
                var partialResult = new T[chunk];
```

Reply:
Oh! I got this, and I will check all changed codes for this.